superset/reports/commands/execute.py

```diff
@@ -526,7 +527,7 @@ def next(self) -> None:
                     return
             self.send()
             self.update_report_schedule_and_log(ReportState.SUCCESS)
-        except CommandException as first_ex:
+        except Exception as first_ex:
```

Comment:
Are we changing these more general exceptions because the CommandException does not return the full trace?

Reply:
rather because all of the exceptions here may not inherit from CommandException since I'm raising them all individually.